EDMarketConnector.py

```diff
@@ -107,7 +107,7 @@
 
     parser.add_argument(
         '--trace-on',
-        help='Mark the selected trace logging as active.',
+        help='Mark the selected trace logging as active. * or all will ensure that every possible trace log appears (in the same way as --trace-all)',
```

Comment:
More succinctly:

`help='Mark the selected trace logging as active. "*" or "all" is the same as using --trace-all'`

and might make flake8 happy about the line length as a bonus ;)